src/homepageExperience/components/FeedbackBar.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import {ComponentSize, FlexBox, JustifyContent} from '@influxdata/clockface'
+import {ThumbsDown, ThumbsUp} from './HomepageIcons'
+
+type OwnProps = {
+  wizardEventName: string
+}
+export default class FeedbackBar extends React.Component<OwnProps> {
+  private handleThumbsUpClick = () => {
+    // handle event here
+  }
+  private handleThumbsDownClick = () => {
+    // handle event here
+  }
+  render() {
```

Comment:
@AyeAndy we need to add the amplitude links here.

Reply:
Just to verify though, these thumbs up and down buttons have amplitude tracking with them? We would still need to have these events classified in amplitude

Reply:
I can add them, I just wasn't sure if we wanted to or not with this PR.